judge/forms.py

```diff
@@ -230,6 +232,22 @@ def clean(self):
             raise ValidationError(_('Must specify either totp_token or webauthn_response.'))
 
 
+class WCIPEGMergeRequestForm(Form):
+    handle = CharField(max_length=50, validators=[UnicodeUsernameValidator()])
+
+    def clean_handle(self):
+        username = self.cleaned_data['handle']
+        if not User.objects.filter(username=username, is_active=True).exists():
+            raise ValidationError(_("Account doesn't exist."))
+        user = User.objects.get(username=username)
```

Comment:
Why not just get the profile and catch the exception instead of doing this exists stuff and then fetching the profile?

Reply:
Done.